src/instructies-coordinator.md

```diff
@@ -0,0 +1,73 @@
+
+## Inloggen en account instellen
+
+Je krijgt inloginstructies van de beheerder. Klik rechtsboven op **Inloggen** en vul de gegevens in. Als je al eerder ingelogd hebt, gebruik je het door jou ingestelde wachtwoord.
+
+Wanneer je voor de eerste keer inlogt, stel je je account eerst in voor gebruik. Voer je naam in als er nog geen naam staat, of wijzig je naam als die onjuist is. Kies vervolgens een nieuw wachtwoord en vul dit twee keer in.
+
+![Account instellen](./img/coordinator-account-instellen.png)
+
+## Stembureaus beheren
+
+Als coördinator kun je stembureaus toevoegen, wijzigen en verwijderen. Kijk hiervoor bij **Beheer verkiezingen** onder de betreffende verkiezing en klik op **Stembureaus**. Als er al stembureaus zijn toegevoegd, zie je die hier staan.
+
+![Stembureaus beheren](./img/stembureaus-beheren.png)
+
+### Stembureau toevoegen
+
+- Klik rechts op de knop **+ Stembureau toevoegen**.
+- Voer de gegevens van het stembureau in en geef aan welke soort stembureau het is. Als het aantal kiesgerechtigden van het stembureau bekend is kun je dit invullen, maar dat hoeft niet.
+- Klik op **Opslaan en toevoegen**.
+
+![Stembureau toevoegen](./img/stembureau-toevoegen.png)
+
+### Stembureau wijzigen of verwijderen
+
+- Klik op het stembureau dat je wilt wijzigen of verwijderen.
+- Wijzig het stembureau door de relevante gegevens te veranderen en vervolgens op **Wijzigingen opslaan** te klikken. Klik op **Annuleren** als je de wijzigingen toch niet wilt opslaan.
+- Verwijder het stembureau door onderaan het scherm op **Stembureau verwijderen** te klikken.
+
+![Stembureau wijzigen](./img/stembureau-wijzigen.png)
+
+## Statusoverzicht steminvoer
+
+Op deze pagina zie je in één oogopslag wat de status van de stembureaus is en wat er nog in de werkvoorraad staat. Hier zie je ook of er fouten en waarschuwingen zijn.
+
+![Statusoverzicht steminvoer](./img/statusoverzicht-steminvoer.png)
+
+## Fouten en waarschuwingen
+
+De invoerders krijgen te maken met verschillende fouten en waarschuwingen, en zij overleggen met jou om tot een oplossing te komen. De oplossingen en handelingen voor de fouten en waarschuwingen vind je in het bijbehorende document in de toolkit.
+
+## Verschil tussen eerste en tweede invoer
+
+Wanneer een stembureau twee keer is ingevoerd, kan het voorkomen dat er een verschil is tussen de eerste en tweede invoer. Op het statusoverzicht zie je dit bovenaan staan.
+
+![Verschil tussen eerste en tweede invoer](./img/verschil-eerste-tweede-invoer.png)
+
+Controleer beide invoeren om na te gaan welke invoer overeenkomt met het proces-verbaal. Klik op de correcte invoer en vervolgens op **Opslaan** om deze te bewaren. De andere invoer wordt weggegooid en moet opnieuw gedaan worden. Let op: als je de tweede invoer bewaart, telt deze nu als eerste invoer. De volgende invoerder doet dan automatisch de tweede invoer.
```

Comment:
```suggestion
Controleer beide invoeren om na te gaan welke invoer overeenkomt met het proces-verbaal. Klik op de correcte invoer en vervolgens op **Opslaan** om deze te bewaren. De andere invoer wordt weggegooid en moet opnieuw gedaan worden.
```
Zou dat stuk over tweede invoer wordt eerste invoer weglaten, volgens mij is vooral relevant dat er nog maar 1 invoer over is, en het dus nog een tweede keer moet worden gedaan.